src/io/cscape_result_io.jl

```diff
@@ -778,7 +732,8 @@ function _cscape_relative_cover(datasets::Vector{Dataset})::YAXArray
     # all(diff(cumsum(n_scens)) .== 1)
 
     start_end_pos = _data_position(n_scens)
-    @showprogress desc="Loading datasets" for (ds_idx, dataset) in zip(start_end_pos, datasets)
+    @showprogress desc = "Loading datasets" for (ds_idx, dataset) in
+                                                zip(start_end_pos, datasets)
```

Comment:
@arlowhite Any way to configure formatter to ignore this case? I'd prefer argument assignment style over variable assignment style in this case (i.e., we want `x=100` here, not `x = 100`)

Reply:
Maybe if you want no spaces around all kwargs `=`?
https://domluna.github.io/JuliaFormatter.jl/stable/#whitespace_in_kwargs

Reply:
That might be it!